### PR TITLE
Add note about size aggregate size, closes #50

### LINKED ISSUE #50: Sizing guidelein for aggregates
Where possible we should provide a guideline to the rough size an aggregate function will take. This isn't going to be 100% correct but the information about if an aggregate takes 10B, 10KB, 100KB will be very helpful to users.

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

File: docs/tremor-query/functions/stats.md [hunk 1]
```diff
@@ -3,10 +3,31 @@
 The `stats` module contains functions for aggregating statistical measures
 of various events.
 
+
+## Size
+
+When using stats functions size becomes an important factor. The exact size an window using
```
Comment (mfelsche) [inline docs/tremor-query/functions/stats.md:9]:
```suggestion
When using stats functions size becomes an important factor. The exact size a window using
```

File: docs/tremor-query/functions/stats.md [hunk 2]
```diff
@@ -3,10 +3,31 @@
...
+aggregates  occupies in memory depends on a two main factors:
```
Comment (mfelsche) [inline docs/tremor-query/functions/stats.md:10]:
```suggestion
aggregates occupies in memory depends on two main factors:
```

File: docs/tremor-query/functions/stats.md [hunk 3]
```diff
@@ -3,10 +3,31 @@
...
+
+* The size of the dimension identifier. I.e. if the window is identified by the string "window" it will
+  require that amount of memory related to this. If it is identified by an array of 10.000 elements
+  all reading "window" it will use (about) 10.0000 that size.
```
Comment (mfelsche) [inline docs/tremor-query/functions/stats.md:14]:
```suggestion
  all reading "window" it will use (about) 10.000 times that size.
```

**Review by mfelsche — APPROVED** [review]
some typos, otherwise LGTM

File: docs/tremor-query/functions/stats.md (hunk 1 above)
Comment (Licenser) [reply docs/tremor-query/functions/stats.md]: :+1:

**Review by mfelsche — APPROVED** [review]

**Review by darach — APPROVED** [review]
LGTM

**Pull request merged**